creator-node/src/routes/files.js

```diff
@@ -753,7 +756,7 @@ module.exports = function (app) {
    * @dev This route does not handle responses by design, so we can pipe the gateway response.
    * TODO: It seems like handleResponse does work with piped responses, as seen from the track/stream endpoint.
    */
-  app.get('/ipfs/:dirCID/:filename', getDirCID)
+  app.get(['/ipfs/:dirCID/:filename', '/content/:dirCID/:filename'], getDirCID)
```

Comment:
want to be mindful of this change here, i know old route is not being removed for some time but i think there may be some other places where we need to make this change, like cloudflare caching maybe cc @dmanjunath

Reply:
yeah good point. i made a card on linear to track this switch in libs from ipfs to content and i added the cf cache reminder on there as well